jvm/ml4ir-inference/src/main/scala/ml4ir/inference/tensorflow/utils/ProtobufUtils.scala

```diff
@@ -21,6 +25,15 @@ object FeatureConfig {
   // zero-arg constructor to be nice to Java
   def apply(): FeatureConfig =
     new FeatureConfig(List.empty, List.empty, None, None)
+  def apply(contextFeatures: java.util.List[FeatureField],
+            documentFeatures: java.util.List[FeatureField]) = {
```

Comment:
This should be sequenceFeatures instead of documentFeatures to maintain parity with other code. We are removing the notion of 'Document' in this PR

Reply:
yes good catch